packages/manager/modules/pci/src/projects/new/config/routing.js

```diff
@@ -0,0 +1,44 @@
+import component from './component';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider
+    .state('pci.projects.new.config', {
+      url: '/config',
+      views: {
+        '': component.name,
+
+        'banner@pci.projects.new.config': 'pciProjectNewConfigBanner',
```

Comment:
Why not integrate the component directly instead of using a ui-view ?

Reply:
So that I am abble to use directly the resolves in component bindings instead of having them in html